modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -297,6 +401,18 @@ struct CompareF
 
 using CompareMats = CompareF<cv::Mat, cv::Mat>;
 using CompareScalars = CompareF<cv::Scalar, cv::Scalar>;
+struct CompareForOptFlow {};
```

Comment:
I think this function shouldn't behere and should be in gapi_video_tests_common.hpp

Reply:
Unfortunately, the following structure uses this struct in its constructor so I have to declare this here

Reply:
But I think I've figured out how to handle it. Will change this

Reply:
fixed